gapic/src/main/com/google/gapid/util/Logging.java

```diff
@@ -99,12 +109,20 @@ public static File getLogDir() {
     return logDir.get().isEmpty() ? null : new File(logDir.get());
   }
 
-  public static String getLogMessages() {
-    return memoryHandler.getMessages();
+  public static Log.Message[] getLogMessages() {
+    return buffer.getMessages();
   }
 
   public static void setListener(Runnable listener) {
-    memoryHandler.listener = (listener == null) ? () -> { /* empty */ } : listener;
+    buffer.listener = (listener == null) ? () -> { /* empty */ } : listener;
+  }
+
+  /**
+   * Adds a {@link Log.Message} to the buffer, bypassing the Java {@link LogManager}.
```

Comment:
Yes and no. Eclipse does give me the warning, but it comes straight from javadoc:
```src/main/com/google/gapid/util/Logging.java:145: warning - Tag @link: reference not found: Log.Message```